test/paper-input.html

```diff
@@ -535,11 +501,24 @@
       });
     });
 
-    // TODO(nowaldorf): This currently crashes on Firefox/Safari.
-    a11ySuite('basic');
-    a11ySuite('label');
-    a11ySuite('label-has-value');
-    a11ySuite('error');
+    suite('a11ySuite', function() {
+      // In the 1.x variant we get this false negative on each input.
```

Comment:
Please credit yourself e.g. `NOTE (notwaldorf):` since a lot of this relies on "as far as I can see"

Reply:
yes mum